cloud/pkg/synccontroller/synccontroller.go

```diff
@@ -107,12 +107,14 @@ func (sctl *SyncController) reconcile() {
 	allClusterObjectSyncs, err := sctl.clusterObjectSyncLister.List(labels.Everything())
 	if err != nil {
 		klog.Errorf("Filed to list all the ClusterObjectSyncs: %v", err)
+		return
```

Comment:
Do we need to consider, when `ClusterObjectSync` resource acquisition fails, we should also try to acquire `ObjectSync` for the next step?

Reply:
Actually from the source code of list, it will never return err if selector is labels.Everything(). And if return err, the ret still can contains some objs, so do we need to return here? Also from the point of code style, this is reasonable to return.

ref: https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/client/listers/reliablesyncs/v1alpha1/clusterobjectsync.go#L51

Reply:
I prefer not to add return, even if the returned object is `nil`, the following code can work well.
What's your opinion?
@fisherxu @gy95

Reply:
my concern is that if return error, the return slice result maybe be incomprehensive, and we couldn't use these incorrect values. 
And accoding to the above comment, the err is never meet. and the following code works well,
So we dont need to add return.